apps/token/src/routes/rewards/home/Rewards.graphql

```diff
@@ -20,7 +20,7 @@ fragment DelegationFields on Delegation {
   epoch
 }
 
-query Rewards($partyId: ID!) {
+query Rewards($partyId: ID!, $delegationsPagination: Pagination) {
```

Comment:
question: this says it's of pagination type but as near as I can tell this is provided a numebr?

Reply:
answered in dm